README.md

```diff
@@ -44,6 +45,7 @@ The available modules are:
 - `readonly-collections` [![Maven Central](https://img.shields.io/maven-central/v/com.black-kamelia.sprinkler/readonly-collections)](https://central.sonatype.com/artifact/com.black-kamelia.sprinkler/readonly-collections)
 - `binary-transcoders` [![Maven Central](https://img.shields.io/maven-central/v/com.black-kamelia.sprinkler/binary-transcoders)](https://central.sonatype.com/artifact/com.black-kamelia.sprinkler/binary-transcoders)
 - `utils` [![Maven Central](https://img.shields.io/maven-central/v/com.black-kamelia.sprinkler/utils)](https://central.sonatype.com/artifact/com.black-kamelia.sprinkler/utils)
+- `jvm-bridge` [![Maven Central](https://img.shields.io/maven-central/v/com.black-kamelia.sprinkler/jvm-bridge)](https://central.sonatype.com/artifact/com.black-kamelia.sprinkler/jvm-bridge)
```

Comment:
pull develop into your branch, you'll see that the links now extend to the module name in monospace